tests/installation/livecdreboot.pm

```diff
@@ -81,9 +82,10 @@ sub run() {
             save_screenshot;
         }
         if (get_var("XEN")) {
+            send_key 'up';    # prevent grub2 timeout
```

Comment:
can the both "up" be moved outside of the 'if' between line 73 and line 74 to prevent grub2 timeout in all cases? I checked myself, "up" is a very good choice. I was afraid it would loop over and start from the last item but it does not :-) That is why I normally use "left" as in "backwards"